src/Native/Unix/Common/pal_utilities.h

```diff
@@ -16,6 +16,28 @@
 #include <type_traits>
 #include <unistd.h>
 
+#ifdef DEBUG
+#define assert_err(cond, msg, err) do \
+{ \
+  if(!(cond)) \
+  { \
+    fprintf(stderr, "%s (%d): error %d: %s. %s\n", __FILE__, __LINE__, err, msg, strerror(err)); \
+    assert(cond && "assert_err failed"); \
+  } \
+} while(0)
+#define assert_msg(cond, msg, val) do \
+{ \
+  if(!(cond)) \
+  { \
+    fprintf(stderr, "%s (%d): error %d: %s\n", __FILE__, __LINE__, val, msg); \
```

Comment:
it's error so it matches the msbuild format (either error or warning). the assert does represent an error -- just doens't have an error code